solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -349,7 +369,8 @@ contract RandomBeacon is IRandomBeacon, Ownable {
         unauthorizedSigningNotificationRewardMultiplier = 50;
         dkgMaliciousResultNotificationRewardMultiplier = 100;
         // slither-disable-next-line too-many-digits
-        authorization.setMinimumAuthorization(100000 * 1e18);
+        authorization.setMinimumAuthorization(100000e18); // 100k T
+        authorization.setAuthorizationDecreaseDelay(403200); // ~10 weeks assuming 15s block time
```

Comment:
Let's confirm the values are correct.

Reply:
I believe it's already there:
https://github.com/keep-network/keep-core/blob/ea31a1ca11b6d78aec3e7cbe50b3ed145c904803/solidity/random-beacon/contracts/RandomBeacon.sol#L397-L417